Pod/Classes/LSTracer.m

```diff
@@ -24,7 +24,7 @@
 @implementation LSTracer {
     NSString* m_accessToken;
     UInt64 m_runtimeGuid;
-    NSDictionary<NSString*, NSString*>* m_tracerJSON;
+    NSDictionary<NSString*, NSObject *>* m_tracerJSON;
```

Comment:
Ah :)   I misremembered which member this is. Generating JSON is such a bummer. Thanks.